Core/UserText.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UserText.swift
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+
+import Foundation
+
+public struct UserText {
+    public static let webPageFailedLoad = NSLocalizedString("web.page.load.failed", comment: "DuckDuckGo could not load the page because...")
```

Comment:
Oh just noticed this! Is it possible to have the Localizable.string and UserText scoped together the same project? There's something strange about this class here but the strings locally